feature/watch.go

```diff
@@ -91,5 +128,33 @@ func Watch(ctx context.Context, path string) error {
 		}
 	}()
 
+	watching = true
+	watchedFile = path
+
+	return nil
+}
+
+func writeBack() error {
+	watchM.Lock() // needed because we are going to access watchedFile
+	defer watchM.Unlock()
+
+	inst.m.RLock()
+	defer inst.m.RUnlock()
+
+	data, err := json.MarshalIndent(&inst.features, "", "  ")
+	if err != nil {
+		return err
+	}
+
+	f, err := os.OpenFile(watchedFile, os.O_TRUNC|os.O_RDWR, 0644)
```

Comment:
Note: this retains permission bits if the file already exists (which it should); only if the config vanishes for some reason, at which point we'll create it with `0644`.

We _might_ want to grab the permission bits off the file on startup and always use that to recreate, or maybe it's not a huge deal.